src/Transition/InputRulesFiles/Rules24-1-0-to-24-2-0.md

```diff
@@ -50,7 +50,8 @@ There are three new optional heat recovery fields at the end which transition wi
 
 # Object Change: ObjectStartsWithN
 
-# Object Change: ObjectStartsWithO
+# epJSON Field Name Change: Output:IlluminanceMap
+`zone_name` --> `zone_or_space_name`
```

Comment:
:+1:

(And no transition (fortran) needed because field didn't change location, and will continue to work)